packages/gatsby-theme-carbon/src/components/AccordionItem/Accordion.module.scss

```diff
@@ -0,0 +1,85 @@
+:global {
+  @import "~carbon-components/scss/components/accordion/accordion";
+}  
+
+.gtc__title {
```

Comment:
Since the classes are generated programattically you don't have to prefix them.

Reply:
Understood. My goal is to avoid was a) to avoid a name collision with 'title' and b) to make it clear where this class was coming from. I can rename the above and remove the notion of prefixes from the other classes.

Reply:
Should be resolved now.